.gitmodules

```diff
@@ -1,4 +1,4 @@
-[submodule "lib/srslte"]
-	path = lib/srslte
-	url = ../srsLTE.git
-	branch = master
```

Comment:
Currently, needs to be changed to
````
[submodule "lib/srsran"]
	path = lib/srsran
	url = ../srsLTE.git
	branch = fembms
````

Shall we rename our fork srslte to srsran?